lib/cache/Cargo.toml

```
@@ -25,6 +25,8 @@ wasmer-engine-universal = { path = "../engine-universal", version = "=2.2.1" }
 wasmer-engine-dylib = { path = "../engine-dylib", version = "=2.2.1" }
 
 [features]
+default = ["wasmer/js-serializable-module"]
```

We should make filesystem also the default to make sure we don't incur in breaking changes for people using the wasmer-cache crate.

```suggestion
default = ["wasmer/js-serializable-module", "filesystem"]
```